.circleci/config.templ.yml

```diff
@@ -141,12 +141,22 @@ commands:
             - start_docker_services:
                 env: SNAPSHOT_CI=1
                 services: testagent << parameters.docker_services >>
+            - run: ulimit -c unlimited
             - run:
                 environment:
                   DD_TRACE_AGENT_URL: << parameters.trace_agent_url >>
                   RIOT_RUN_RECOMPILE_REQS: "<< pipeline.parameters.riot_run_latest >>"
                 command: |
                   ./scripts/run-test-suite '<<parameters.pattern>>' <<pipeline.parameters.coverage>> 1
+            - run:
+                command: |
+                  mkdir -p /tmp/core_dumps
+                  cp core.* /tmp/core_dumps
+                  ./scripts/bt
+                when: on_fail
+            - store_artifacts:
+                name: "Store core dumps"
+                path: /tmp/core_dumps
```

Comment:
I like this, but we'd need some way to also extract the precise versions of system and artifact .so files (or at least build things in a reliable way).

Please ignore if the intent isn't to extract corefiles for further analysis.

Reply:
Good point. I had this here from earlier experiments. But indeed if we want to keep these, we should collect all the binaries as well. Another complication might be the platform, meaning the analysis might have to be done on a similar architecture